ci/sign-all-tasks.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
```

Comment:
What is the output of this? Are the tasks pushed as artifacts like they are today but now they're signed? That is probably ideal.

Reply:
Currently, all I am doing is creating the zip files and sending them off for signing. They are returned to the same location with the signed zip. I am not yet publishing them as artifacts. It also appears that we already create nuget packages of individual tasks in some cases, but I wasn't sure when this happens versus creating the one zip. I think we also have a problem with actual nuget packaging, right?

Reply:
@jpricket Have we confirmed that signed versions of the packages are getting pushed?

Reply:
I confirmed that the files are signed and updated in place, but I have not confirmed that the published ones are still signed. I will have to confirm that after I push the changes and get a real CI build.